crates/server/src/bin/warg-server.rs

```diff
@@ -52,7 +58,27 @@ async fn main() -> Result<()> {
     tracing::debug!("args: {args:?}");
 
     // TODO: pull the signing key from the system keyring
-    let operator_key = std::env::var("WARG_DEMO_OPERATOR_KEY")?.parse()?;
+    // TODO: Add --warg-operator-key[-file] and others cli options
+    let operator_key: PrivateKey = match std::env::var("WARG_OPERATOR_KEY_FILE") {
```

Comment:
Extract into a function named e.g.  `get_operator_key`.

Reply:
I extracted a function `get_operator_key` from which I also extracted the arg (file)/env processing behavior into a function `get_opt_content` in a crate `warg-args-util` which can also be picked up by the `warg` CLI client or other apps if desired. Used `get_opt_content` to also pick up the database URL from a file since it contains sensitive data.

Reply:
I think splitting it out into its own entire crate is premature. We should probably wait to do that in a PR wherein we actually want to call this function from somewhere else.

Reply:
Added patch to undo the new library creation.